src/admin-views/meta-box.php

```diff
@@ -105,147 +119,157 @@
+							</span>
+						</div>
+						<div class="input_block">
+							<label class="ticket_form_label" for="ticket_end_date"><?php esc_html_e( 'End sale:', 'event-tickets' ); ?></label>
+							<input autocomplete="off" type="text" class="ticket_field" size='10' name="ticket_end_date" id="ticket_end_date" value="">
 
-	<?php // the settings panel ?>
-	<div id="panel_settings" class="ticket_panel panel_settings">
-
-		<?php if ( $show_global_stock ) : ?>
-			<div id="tribe-global-stock-settings" class="event-wide-settings">
-				<div class="eventtable ticket_list eventForm">
-					<div>
-						<label for="tribe-tickets-enable-global-stock">
-							<?php esc_html_e( 'Enable global stock', 'event-tickets' ); ?>
-						</label>
-						<input type="checkbox" name="tribe-tickets-enable-global-stock" id="tribe-tickets-enable-global-stock" value="1" <?php checked( $global_stock->is_enabled() ); ?> />
-					</div>
-					<div id="tribe-tickets-global-stock-level">
-							<label for="tribe-tickets-global-stock">
-								<?php esc_html_e( 'Global stock level', 'event-tickets' ); ?>
-							</label>
-							<input type="number" name="tribe-tickets-global-stock" id="tribe-tickets-global-stock" value="<?php echo esc_attr( $global_stock->get_stock_level() ); ?>" />
-							<span class="tribe-tickets-global-sales">
-								<?php echo esc_html( sprintf( _n( '(%s sold)', '(%s sold)', $global_stock->tickets_sold(), 'event-tickets' ), $global_stock->tickets_sold() ) ); ?>
+							<span class="ticket_end_time ticket_time">
+								<?php echo tribe_get_datetime_separator(); ?>
+								<select name="ticket_end_hour" id="ticket_end_hour" class="ticket_field tribe-dropdown">
+									<?php echo $endHourOptions; ?>
+								</select>
+								<select name="ticket_end_minute" id="ticket_end_minute" class="ticket_field tribe-dropdown">
+									<?php echo $endMinuteOptions; ?>
+								</select>
+								<?php if ( ! strstr( get_option( 'time_format', Tribe__Date_Utils::TIMEFORMAT ), 'H' ) ) : ?>
+									<select name="ticket_end_meridian" id="ticket_end_meridian" class="ticket_field tribe-dropdown">
+										<?php echo $endMeridianOptions; ?>
+									</select>
+								<?php endif; ?>
 							</span>
+
+							<p class="description">
+								<?php esc_html_e( 'When will ticket sales occur?', 'event-tickets' ); ?>
+								<?php
+								// Why break in and out of PHP? because I want the space between the phrases without including them in the translations
+								if ( class_exists( 'Tribe__Events__Main' ) && Tribe__Events__Main::POSTTYPE === get_post_type( $post ) ) {
+									esc_html_e( "If you don't set a start/end date for sales, tickets will be available from now until the event ends.", 'event-tickets' );
+								}
+								?>
+							</p>
+						</div>
+						<?php
+						/**
+						 * Allows for the insertion of additional content into the ticket edit form - advanced section
+						 *
+						 * @var Post ID
+						 * @var null Ticket ID
```

Comment:
Switch these `@var` tags to `@param` tags ... is `$ticket_id` here despite always being null for back-compat reasons? Might be worth noting as much if so